procedures/foreman/release.md.erb

```diff
@@ -1,7 +1,15 @@
 [ ] Make this post a wiki
 
+## Roles
+
+* Release Owner: <%= owner %>
+* Release Engineer: <%= engineer %>
+* Installer Maintainer: @
+
 # Manual updates: <%= target_date %>
 
+## Release Engineer
```

Comment:
Should these headers repeat the username?

Reply:
Interesting idea.